pkg/providers/vsphere/config/template-cp.yaml

```diff
@@ -452,6 +452,7 @@ spec:
     name: cloud-provider-vsphere-credentials
   - kind: ConfigMap
     name: cpi-manifests
+  strategy: Reconcile
```

Comment:
CRS.spec.strategy is immutable, so this is not gonna work
We need to either delete the CRS before we apply the spec or just use a different name for the reconcile CRS so there is no collision. If we decide to use a new name maybe we should add something to clean up the old one. Although we can do as a follow up.

Reply:
Updated the name to be `<clusterName>-crs` instead of `<clusterName>-crs-0`. Will do the clean up in a follow up

Reply:
Cool
Now that I think about it, what do you think about `<clusterName>-cpi-csi`?
Since that's what this crs manages

Reply:
yes, i like that! updated it